content/en/docs/reference/command-line-tools-reference/feature-gates/apparmor-fields.md

```diff
@@ -0,0 +1,13 @@
+---
+title: AppArmorFields
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.30"
+---
+Enable AppArmor related security context settings.
```

Comment:
Could make security context a hyperlink or tooltip, but it's better to have something than nothing.